packages/core/src/RepoSync.ts

```diff
@@ -74,11 +74,11 @@ export class RepoSync {
   }
 
   getClocks() {
-    // const ours = EMPTY_CLOCKMAP
+    const ours = EMPTY_CLOCKMAP
     const theirs = EMPTY_CLOCKMAP
 
-    const persistedClock = localStorage.getItem('ourClock')
-    const ours = persistedClock ? (Map(JSON.parse(persistedClock)) as ClockMap) : EMPTY_CLOCKMAP
+    // const persistedClock = localStorage.getItem('ourClock')
+    // const ours = persistedClock ? (Map(JSON.parse(persistedClock)) as ClockMap) : EMPTY_CLOCKMAP
```

Comment:
Is this a failed experiment or something disabled temporarily?

Reply:
it's work in progress that leaked into this branch